projects/libra/project.yaml

```diff
@@ -2,3 +2,7 @@ homepage: "https://github.com/libra/libra"
 primary_contact: "davidwong.crypto@gmail.com"
 auto_ccs:
   - "davidwg@fb.com"
+sanitizers:
+  - address
+fuzzing_engines:
+  - libfuzzer
```

Comment:
+1 please add `language: rust`, and we should think whether we can automatically catch non C/C++ projects without `language` attribute